src/etc/inc/openvpn.inc

```diff
@@ -1007,8 +1007,16 @@ function openvpn_reconfigure($mode, $settings) {
 		case 'server_tls':
 		case 'server_tls_user':
 		case 'server_user':
-			$ca = lookup_ca($settings['caref']);
-			openvpn_add_keyfile($ca['crt'], $conf, $mode_id, "ca");
+			// ca_chain() expects parameter to be passed  by reference. 
+			// avoid passing the whole settinsg array, as param names or 
```

Comment:
s/settinsg/settings/
And the line above has 2 spaces between "passed" and "by".